src/modules/Instabug.ts

```diff
@@ -547,18 +550,66 @@ export const addExperiments = (experiments: string[]) => {
 /**
  * Remove experiments from next report.
  * @param experiments An array of experiments to remove from the next report.
+ *
+ * @deprecated Please migrate to the new Feature Flags APIs: {@link removeFeatureFlags}.
  */
 export const removeExperiments = (experiments: string[]) => {
   NativeInstabug.removeExperiments(experiments);
 };
 
 /**
  * Clear all experiments
+ *
+ * @deprecated Please migrate to the new Feature Flags APIs: {@link removeAllFeatureFlags}.
  */
 export const clearAllExperiments = () => {
   NativeInstabug.clearAllExperiments();
 };
 
+/**
+ * Add feature flags to the next report.
+ * @param featureFlags An array of feature flags to add to the next report.
+ */
+export const addFeatureFlags = (featureFlags: FeatureFlag[]) => {
+  const flags: Record<string, string | undefined> = {};
+
+  for (const item of featureFlags) {
+    flags[item.name] = item.variant ? item.variant : '';
+  }
```

Comment:
I think using `Object.fromEntries` would be a more declarative, thus readable, way to create the `flags` object, what do you think?

```suggestion
    const entries = featureFlags.map((item) => [item.name, item.variant || '']);
    const flags = Object.fromEntries(entries);
```